web/src/containers/QuarterlyBudgetSummary.js

```diff
@@ -7,150 +7,130 @@ import { t } from '../i18n';
+              <colgroup>
+                <col className="table-cms--col-header__fixed-width" />
+                <col />
+                <col />
+                <col />
+                <col />
+              </colgroup>
+              <thead>
+                <tr>
+                  <th
+                    key={year}
+                    className="center"
+                    id={`quarterly-budget-summary-${source}-fy-${year}`}
+                  >
+                    {t('ffy', { year })}
+                  </th>
+                  {QUARTERS.map(q => (
                     <th
+                      key={q}
                       className="center"
-                      id={`quarterly-budget-summary-${source}-total`}
+                      id={`quarterly-budget-summary-${source}-fy-${year}-q${q}`}
                     >
-                      {t('table.total')}
+                      {t('table.quarter', { q })}
                     </th>
-                  </tr>
-                  <tr>
-                    <th id={`quarterly-budget-summary-${source}-null2`} />
-                    {years.map((year, i) => (
-                      <Fragment key={year}>
-                        {QUARTERS.map(q => (
-                          <th
-                            key={q}
-                            className="center"
-                            id={`quarterly-budget-summary-${source}-fy-${year}-q${q}`}
-                          >
-                            {t('table.quarter', { q })}
-                          </th>
-                        ))}
-                        <th
-                          className={`right-align ${color(i)}-light`}
-                          id={`quarterly-budget-summary-${source}-fy-${year}-subtotal`}
-                        >
-                          {t('table.subtotal')}
-                        </th>
-                      </Fragment>
-                    ))}
+                  ))}
+                  <th
+                    className="right-align"
+                    id={`quarterly-budget-summary-${source}-fy-${year}-subtotal`}
+                  >
+                    {t('table.subtotal')}
+                  </th>
+                </tr>
+              </thead>
+              <tbody>
+                {Object.keys(EXPENSE_NAME_DISPLAY).map(name => (
+                  <tr
+                    key={name}
+                  >
                     <th
-                      className="bg-gray-light"
-                      id={`quarterly-budget-summary-${source}-total2`}
-                    />
-                  </tr>
-                </thead>
-                <tbody>
-                  {Object.keys(EXPENSE_NAME_DISPLAY).map(name => (
-                    <tr
-                      key={name}
-                      className={`${name === 'combined' ? 'bold' : ''}`}
+                      headers={`quarterly-budget-summary-${source}-null1 quarterly-budget-summary-${source}-null2`}
```

Comment:
Does `headers` apply to `th` elements?  This may fall more under the purview of #1459, not sure.

Reply:
I just now ripped these out and did my best approximation of the structure #1459 will move us towards. 🤷‍♀️